packages/one-app-bundler/package-lock.json

```diff
@@ -4,14 +4,6 @@
 	"lockfileVersion": 1,
 	"requires": true,
 	"dependencies": {
-		"@americanexpress/eslint-plugin-one-app": {
```

Comment:
@10xLaCroixDrinker why did this get removed?

Reply:
lerna removed it